src/containers/TexterTodo.jsx

```diff
@@ -240,22 +245,23 @@ const queries = {
       console.log("TexterTodo ownProps", ownProps);
       // based on ?review=1 in location.search
       // exclude isOptedOut: false, validTimezone: true
-      const contactsFilter = ownProps.location.query.review
-        ? {
-            messageStatus: ownProps.messageStatus,
-            errorCode: ["0"],
-            ...(ownProps.params.reviewContactId && {
-              contactId: ownProps.params.reviewContactId
-            })
-          }
-        : {
-            messageStatus: ownProps.messageStatus,
-            ...(!ownProps.params.reviewContactId && { isOptedOut: false }),
-            ...(ownProps.params.reviewContactId && {
-              contactId: ownProps.params.reviewContactId
-            }),
-            validTimezone: true
-          };
+      const contactsFilter =
+        ownProps.location.query.review === "1"
```

Comment:
This isn't related to this change -- it should have been here before and I wanted to fix it

Reply:
(the check for `=== "1"`)